allennlp_models/generation/dataset_readers/cnn_dm.py

```diff
@@ -157,15 +159,21 @@ def text_to_instance(
         if self._source_max_tokens is not None and len(tokenized_source) > self._source_max_tokens:
             tokenized_source = tokenized_source[: self._source_max_tokens]
 
-        source_field = TextField(tokenized_source, self._source_token_indexers)
+        source_field = TextField(tokenized_source)
```

Comment:
I'm digging this, the fact that you don't even specify an indexer here anymore. That's where we want to go with the "Instances are nothing but data." concept.